src/User32.Desktop/User32.cs

```diff
@@ -647,13 +1103,71 @@ public static partial class User32
         [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
         public static extern int RegisterWindowMessage(string lpString);
 
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool GetWindowDisplayAffinity(IntPtr hWnd, out int dwAffinity);
+
+        [DllImport(nameof(User32), SetLastError = true)]
+        public static extern bool EnumDisplayMonitors(IntPtr hdc, IntPtr lprcClip, MonitorEnumDelegate lpfnEnum, IntPtr dwData);
+
+        [DllImport(nameof(User32), SetLastError = true, CharSet = CharSet.Unicode)]
+        public static extern bool GetClassInfoEx(IntPtr hInstance, string lpClassName, ref WNDCLASSEX lpWndClass);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
+        public static extern unsafe bool GetMonitorInfo(
+            IntPtr hMonitor,
+            [Friendly(FriendlyFlags.Out | FriendlyFlags.Optional)] MONITORINFO* lpmi);
+
+        [DllImport(nameof(User32), EntryPoint = "GetMonitorInfo", SetLastError = true)]
```

Comment:
As odd as it may seem, I thought this way would be better than having two overloads. Also I think that the generator tries to create two identical methods for the ` IntPtr`  overloads and it breaks. Let's see if my memory is not playing tricks on me.